dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -59,6 +59,11 @@ internal.error.CWMMC0006E=CWMMC0006E: An internal error occurred.
 internal.error.CWMMC0006E.explanation=The application server experienced an internal error.
 internal.error.CWMMC0006E.useraction=Gather a set of traces and open a new issue against your application server.
 
+#Information regarding libraryRef attribute taking effect only after server restart
+configurationChange.info.CWMMC0007I=CWMMC0007I: A configuration change was detected for the MicroProfile Metrics feature. If a changes was made to the libraryRef attribute then a server restart is required for the changes to take effect.
+configurationChange.info.CWMMC0007I.explanation=A configuration change was detected for the MicroProfile Metrics feature. If a changes was made to the libraryRef attribute then a server restart is required for the changes to take effect.
```

Comment:
a changes -> a change
the changes -> the change

Reply:
explanation should say something about how the mpMetrics feature does not allow for dynamic replacement of the library as applications could have registered metrics at app startup time.